time stamp		# time since system start (microseconds)


bool start_planning		# True when mission path is occupied 
float64[] target_position	# Position of the next (safe) incremental WP


